FusionIIIT/applications/counselling_cell/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
 
+from applications.academic_information.models import Student
 # Create your models here.
+
+
+POSTIONS= (
+    ('pos1', 'Student'),
```

Comment:
Make the first element of all tuples meaningful and second element displayable (Student_Guide is not displayable, Student Guide is).

For ex., first element can be `stu_guide` and second tuple `Student Guide`.